srcpkgs/cargo-auditable/template

```diff
@@ -1,15 +1,18 @@
 # Template file for 'cargo-auditable'
+# Keep synced with cargo-auditable-bootstrap
 pkgname=cargo-auditable
 version=0.5.2
 revision=1
```

Comment:
let's revbump this, so it's auditable too

Reply:
good catch